test/support/utils.rb

```diff
@@ -74,21 +89,21 @@ def debug_on_tcpip repl_prompt = '(rdbg:remote)'
     end
 
     def setup_remote_debuggee(cmd)
-      @remote_r, @remote_w, @remote_debuggee_pid = PTY.spawn(cmd, :in=>'/dev/null', :out=>'/dev/null')
+      @remote_r, @remote_w, @remote_debuggee_pid = PTY.spawn(cmd)
```

Comment:
I need to change this part to output complete backlog(In the above example, this would be `"hello"`) in remote deguggee.